### PR TITLE
Fix StyleCop warnings in OpenRA.Game

### PR DESCRIPTION
**Pull request by Hellhake**

### CODE REVIEW

File: OpenRA.Game/Graphics/WorldRenderer.cs [hunk 1]
```diff
@@ -92,7 +92,7 @@ List<IRenderable> GenerateRenderables()
 			if (world.OrderGenerator != null)
 				effectRenderables = effectRenderables.Concat(world.OrderGenerator.RenderAfterWorld(this, world));
 
-			// Iterating via foreach() copies the structs, so enumerate by index
+			// Iterating via foreach () copies the structs, so enumerate by index
```
Comment (RoosterDragon) [inline OpenRA.Game/Graphics/WorldRenderer.cs]: This is silly, maybe just drop the brackets.

File: OpenRA.Game/Input/Keycode.cs [hunk 2]
```diff
@@ -419,8 +419,8 @@ public static class KeycodeExts
 			{ Keycode.DECIMALSEPARATOR, "DecimalSeparator" },
 			{ Keycode.CURRENCYUNIT, "CurrencyUnit" },
 			{ Keycode.CURRENCYSUBUNIT, "CurrencySubUnit" },
-			{ Keycode.KP_LEFTPAREN, "Keypad (" },
-			{ Keycode.KP_RIGHTPAREN, "Keypad )" },
+			{ Keycode.KP_LEFTPAREN, "Keypad(" },
+			{ Keycode.KP_RIGHTPAREN, "Keypad)" },
```
Comment (RoosterDragon) [inline OpenRA.Game/Input/Keycode.cs]: This probably shouldn't have been changed.

File: OpenRA.Game/Exts.cs [hunk 3]
```diff
@@ -375,6 +373,7 @@ public static T[] MakeArray<T>(int count, Func<int, T> f)
 			var result = new T[width, height];
 			for (var i = 0; i < width; i++)
 				for (var j = 0; j < height; j++)
+
```
Comment (pchote) [inline OpenRA.Game/Exts.cs]: This newline doesn't look right.

File: OpenRA.Game/FileFormats/Blast.cs [hunk 4]
```diff
@@ -142,9 +142,9 @@ public static byte[] Decompress(byte[] src)
 						}
 					} while (len != 0);
 				}
-				// literal value
 				else
-				{
+					{
```
Comment (pchote) [inline OpenRA.Game/FileFormats/Blast.cs]: Looks like you accidentally indented the brace too.

File: OpenRA.Game/Player.cs [hunk 5]
```diff
@@ -21,14 +21,11 @@
...
 {
-	public enum PowerState { Normal, Low, Critical };
-	public enum WinState { Undefined, Won, Lost };
+	public enum PowerState { Normal, Low, Critical }
+	public enum WinState { Undefined, Won, Lost }
 
-	public class Player :  IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding
+	public class Player : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding
 	{
-		public Actor PlayerActor;
```
Comment (pchote) [inline OpenRA.Game/Player.cs]: This should be made `readonly`, and then stay here at the top of the class.

File: OpenRA.Game/Support/PerfTimer.cs [hunk 6]
```diff
@@ -28,9 +31,6 @@ public sealed class PerfTimer : IDisposable
 		static ThreadLocal<PerfTimer> Parent = new ThreadLocal<PerfTimer>();
 
 		// Tree settings
```
Comment (pchote) [inline OpenRA.Game/Support/PerfTimer.cs]: This comment and the FormatString need to move together with the other three.

**Comment by pchote** [comment]
:+1: after fixing the above nits.

**Comment by Hellhake** [comment]
All issues should be resolved now.

**Comment by reaperrr** [comment]
@Hellhake: one of @pchote's nits is still showing for me (```PlayerActor``` in ```Player.cs``` should be made ```readonly``` and stay on top of the class)

**Comment by obrakmann** [comment]
He did that, you just can't see it because pchote commented on the removed line, not on the added one.

**Comment by reaperrr** [comment]
Sorry, my bad.

**Comment by obrakmann** [comment]
I still get 202 violations in OpenRA.Game, but all of those are about variable naming issues, which would probably lead to a much bigger diff if they were fixed all at once.

So this is a good start, :+1:

**Comment by Hellhake** [comment]
@obrakmann I did this on purpose because

a) at the time i created the fixes i did not know if its appreciated to make real changes in the code just to make StyleCop happy
b) i didn't know if this would break something
c) i wanted to separate formatting changes from real changes
d) one change of a variable name may affect multiple projects. Rebasing this could be very hard. Therefore i wanted to keep the new pr as small as possible.

However i will try to fix the naming issues in a another pr :)

**Comment by Mailaender** [comment]
I filed https://github.com/OpenRA/OpenRA/issues/7251 which was a pretty ugly job.

**Pull request merged**